### ISSUE TITLE
Xcode issue Framework Flutter Not Found even though I have the executable file for flutter and and I have a path for it in my ./zshrc

### ISSUE THREAD

**Issue by dm9375** [flutter/flutter#70476]
Hi I'm trying to start my first flutter personal project in VS code but the simulator isn't building, I'm so confused and tried to do flutter clean but it's still not working. Also, everything checks out when I do flutter doctor. Please help!

<details> 
<summary>logs</summary> 

```bash
renatabuczkowska@MacBook-Air renata_app % flutter clean
Cleaning Xcode workspace...                                        21.9s
Deleting build...                                                  408ms
Deleting .dart_tool...                                              56ms
Deleting Generated.xcconfig...                                       1ms
Deleting flutter_export_environment.sh...                            1ms
Deleting App.framework...                                          107ms
renatabuczkowska@MacBook-Air renata_app % flutter run   
Running "flutter pub get" in renata_app...                          0.8s
Launching lib/main.dart on iPhone 12 Pro Max in debug mode...
 
Running Xcode build...                                                  
 └─Compiling, linking and signing...                         5.9s
Xcode build done.                                           36.5s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    While building module 'Flutter' imported from
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:8:10:
    warning: double-quoted include "FlutterAppDelegate.h" in framework header, expected
    angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterAppDelegate.h"
             ^~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterAppDelegate.h>
    While building module 'Flutter' imported from
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:8:
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterAppDelegat
    e.h:10:10: warning: double-quoted include "FlutterMacros.h" in framework header,
    expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterMacros.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterMacros.h>
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterAppDelegat
    e.h:11:10: warning: double-quoted include "FlutterPlugin.h" in framework header,
    expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterPlugin.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterPlugin.h>
    While building module 'Flutter' imported from
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:8:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterAppDelegat
    e.h:11:
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterPlugin.h:1
    1:10: warning: double-quoted include "FlutterBinaryMessenger.h" in framework header,
    expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterBinaryMessenger.h"
             ^~~~~~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterBinaryMessenger.h>
    While building module 'Flutter' imported from
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:8:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterAppDelegat
    e.h:11:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterPlugin.h:1
    1:
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterBinaryMess
    enger.h:10:10: warning: double-quoted include "FlutterMacros.h" in framework header,
    expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterMacros.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterMacros.h>
    While building module 'Flutter' imported from
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:8:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterAppDelegat
    e.h:11:
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterPlugin.h:1
    2:10: warning: double-quoted include "FlutterChannels.h" in framework header,
    expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterChannels.h"
             ^~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterChannels.h>
    While building module 'Flutter' imported from
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:8:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterAppDelegat
    e.h:11:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterPlugin.h:1
    2:
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterChannels.h
    :8:10: warning: double-quoted include "FlutterBinaryMessenger.h" in framework header,
    expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterBinaryMessenger.h"
             ^~~~~~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterBinaryMessenger.h>
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterChannels.h
    :9:10: warning: double-quoted include "FlutterCodecs.h" in framework header, expected
    angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterCodecs.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterCodecs.h>
    While building module 'Flutter' imported from
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:8:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterAppDelegat
    e.h:11:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterPlugin.h:1
    2:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterChannels.h
    :9:
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterCodecs.h:9
    :10: warning: double-quoted include "FlutterMacros.h" in framework header, expected
    angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterMacros.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterMacros.h>
    While building module 'Flutter' imported from
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:8:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterAppDelegat
    e.h:11:
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterPlugin.h:1
    3:10: warning: double-quoted include "FlutterCodecs.h" in framework header, expected
    angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterCodecs.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterCodecs.h>
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterPlugin.h:1
    4:10: warning: double-quoted include "FlutterPlatformViews.h" in framework header,
    expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterPlatformViews.h"
             ^~~~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterPlatformViews.h>
    While building module 'Flutter' imported from
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:8:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterAppDelegat
    e.h:11:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterPlugin.h:1
    4:
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterPlatformVi
    ews.h:10:9: warning: double-quoted include "FlutterCodecs.h" in framework header,
    expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #import "FlutterCodecs.h"
            ^~~~~~~~~~~~~~~~~
            <Flutter/FlutterCodecs.h>
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterPlatformVi
    ews.h:11:9: warning: double-quoted include "FlutterMacros.h" in framework header,
    expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #import "FlutterMacros.h"
            ^~~~~~~~~~~~~~~~~
            <Flutter/FlutterMacros.h>
    While building module 'Flutter' imported from
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:8:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterAppDelegat
    e.h:11:
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterPlugin.h:1
    5:10: warning: double-quoted include "FlutterTexture.h" in framework header, expected
    angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterTexture.h"
             ^~~~~~~~~~~~~~~~~~
             <Flutter/FlutterTexture.h>
    While building module 'Flutter' imported from
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:8:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterAppDelegat
    e.h:11:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterPlugin.h:1
    5:
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterTexture.h:
    11:10: warning: double-quoted include "FlutterMacros.h" in framework header, expected
    angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterMacros.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterMacros.h>
    While building module 'Flutter' imported from
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:9:10:
    warning: double-quoted include "FlutterBinaryMessenger.h" in framework header,
    expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterBinaryMessenger.h"
             ^~~~~~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterBinaryMessenger.h>
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:10:10:
    warning: double-quoted include "FlutterCallbackCache.h" in framework header, expected
    angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterCallbackCache.h"
             ^~~~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterCallbackCache.h>
    While building module 'Flutter' imported from
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:10:
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterCallbackCa
    che.h:10:10: warning: double-quoted include "FlutterMacros.h" in framework header,
    expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterMacros.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterMacros.h>
    While building module 'Flutter' imported from
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:11:10:
    warning: double-quoted include "FlutterChannels.h" in framework header, expected
    angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterChannels.h"
             ^~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterChannels.h>
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:12:10:
    warning: double-quoted include "FlutterCodecs.h" in framework header, expected
    angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterCodecs.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterCodecs.h>
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:13:10:
    warning: double-quoted include "FlutterDartProject.h" in framework header, expected
    angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterDartProject.h"
             ^~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterDartProject.h>
    While building module 'Flutter' imported from
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:13:
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterDartProjec
    t.h:10:10: warning: double-quoted include "FlutterMacros.h" in framework header,
    expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterMacros.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterMacros.h>
    While building module 'Flutter' imported from
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:14:10:
    warning: double-quoted include "FlutterEngine.h" in framework header, expected
    angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterEngine.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterEngine.h>
    While building module 'Flutter' imported from
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:14:
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterEngine.h:1
    1:10: warning: double-quoted include "FlutterBinaryMessenger.h" in framework header,
    expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterBinaryMessenger.h"
             ^~~~~~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterBinaryMessenger.h>
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterEngine.h:1
    2:10: warning: double-quoted include "FlutterDartProject.h" in framework header,
    expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterDartProject.h"
             ^~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterDartProject.h>
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterEngine.h:1
    3:10: warning: double-quoted include "FlutterMacros.h" in framework header, expected
    angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterMacros.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterMacros.h>
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterEngine.h:1
    4:10: warning: double-quoted include "FlutterPlugin.h" in framework header, expected
    angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterPlugin.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterPlugin.h>
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterEngine.h:1
    5:10: warning: double-quoted include "FlutterTexture.h" in framework header, expected
    angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterTexture.h"
             ^~~~~~~~~~~~~~~~~~
             <Flutter/FlutterTexture.h>
    While building module 'Flutter' imported from
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:15:10:
    warning: double-quoted include "FlutterHeadlessDartRunner.h" in framework header,
    expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterHeadlessDartRunner.h"
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterHeadlessDartRunner.h>
    While building module 'Flutter' imported from
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:15:
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterHeadlessDa
    rtRunner.h:10:10: warning: double-quoted include "FlutterBinaryMessenger.h" in
    framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterBinaryMessenger.h"
             ^~~~~~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterBinaryMessenger.h>
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterHeadlessDa
    rtRunner.h:11:10: warning: double-quoted include "FlutterDartProject.h" in framework
    header, expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterDartProject.h"
             ^~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterDartProject.h>
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterHeadlessDa
    rtRunner.h:12:10: warning: double-quoted include "FlutterEngine.h" in framework
    header, expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterEngine.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterEngine.h>
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterHeadlessDa
    rtRunner.h:13:10: warning: double-quoted include "FlutterMacros.h" in framework
    header, expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterMacros.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterMacros.h>
    While building module 'Flutter' imported from
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:16:10:
    warning: double-quoted include "FlutterMacros.h" in framework header, expected
    angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterMacros.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterMacros.h>
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:17:10:
    warning: double-quoted include "FlutterPlatformViews.h" in framework header, expected
    angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterPlatformViews.h"
             ^~~~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterPlatformViews.h>
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:18:10:
    warning: double-quoted include "FlutterPlugin.h" in framework header, expected
    angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterPlugin.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterPlugin.h>
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:19:10:
    warning: double-quoted include "FlutterPluginAppLifeCycleDelegate.h" in framework
    header, expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterPluginAppLifeCycleDelegate.h"
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterPluginAppLifeCycleDelegate.h>
    While building module 'Flutter' imported from
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:19:
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterPluginAppL
    ifeCycleDelegate.h:8:10: warning: double-quoted include "FlutterPlugin.h" in
    framework header, expected angle-bracketed instead
    [-Wquoted-include-in-framework-header]
    #include "FlutterPlugin.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterPlugin.h>
    While building module 'Flutter' imported from
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:20:10:
    warning: double-quoted include "FlutterTexture.h" in framework header, expected
    angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterTexture.h"
             ^~~~~~~~~~~~~~~~~~
             <Flutter/FlutterTexture.h>
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:21:10:
    warning: double-quoted include "FlutterViewController.h" in framework header,
    expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterViewController.h"
             ^~~~~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterViewController.h>
    While building module 'Flutter' imported from
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Runner/GeneratedPluginRegistrant.h:8:
    In file included from <module-includes>:1:
    In file included from /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/Flutter.h:21:
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterViewContro
    ller.h:11:10: warning: double-quoted include "FlutterBinaryMessenger.h" in framework
    header, expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterBinaryMessenger.h"
             ^~~~~~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterBinaryMessenger.h>
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterViewContro
    ller.h:12:10: warning: double-quoted include "FlutterDartProject.h" in framework
    header, expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterDartProject.h"
             ^~~~~~~~~~~~~~~~~~~~~~
             <Flutter/FlutterDartProject.h>
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterViewContro
    ller.h:13:10: warning: double-quoted include "FlutterEngine.h" in framework header,
    expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterEngine.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterEngine.h>
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterViewContro
    ller.h:14:10: warning: double-quoted include "FlutterMacros.h" in framework header,
    expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterMacros.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterMacros.h>
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterViewContro
    ller.h:15:10: warning: double-quoted include "FlutterPlugin.h" in framework header,
    expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterPlugin.h"
             ^~~~~~~~~~~~~~~~~
             <Flutter/FlutterPlugin.h>
    /Users/renatabuczkowska/Desktop/Personal
    Programming/Renata/renata_app/ios/Flutter/Flutter.framework/Headers/FlutterViewContro
    ller.h:16:10: warning: double-quoted include "FlutterTexture.h" in framework header,
    expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #include "FlutterTexture.h"
             ^~~~~~~~~~~~~~~~~~
             <Flutter/FlutterTexture.h>
    46 warnings generated.
    46 warnings generated.
    ld: framework not found Flutter
    clang: error: linker command failed with exit code 1 (use -v to see invocation)
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description

Could not build the application for the simulator.
Error launching application on iPhone 12 Pro Max.
```

</details>

**Comment by TahaTesser** [comment]
Hi @dm9375 
Can you please provide your `flutter doctor -v` and `flutter run --verbose` logs, and a [minimal complete reproducible code sample](https://stackoverflow.com/help/minimal-reproducible-example)?
Thank you

**Comment by pedromassangocode** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.
